lib/mongo/monitoring/event/command_failed.rb

```diff
@@ -52,16 +59,18 @@ class CommandFailed < Mongo::Event::Base
         # @param [ Integer ] request_id The request id.
         # @param [ Integer ] operation_id The operation id.
         # @param [ String ] message The error message.
+        # @param [ BSON::Document ] failure The error document, if any.
         # @param [ Float ] duration The duration the command took in seconds.
         #
         # @since 2.1.0
-        def initialize(command_name, database_name, address, request_id, operation_id, message, duration)
-          @command_name = command_name
+        def initialize(command_name, database_name, address, request_id, operation_id, message, failure, duration)
```

Comment:
Given that this isn't marked with `@api private`, is it worth putting the new parameter last and adding a default value so that it doesn't break API? An alternative would be to mark this as `@api private`, although that still seems like we'd be making a breaking change.

Reply:
I will mark it private. Can you think of a use case when someone would be creating events outside of the driver?